cmake/HelperFunctions.cmake

```diff
@@ -215,6 +215,16 @@ function(read_yaml FILENAME)
             list(GET ITEM_LIST 1 TEMP_BUILD_FLAGS)
             string(REPLACE "," " " LOC_BUILD_FLAGS "${TEMP_BUILD_FLAGS}")
             set(LOCAL_BUILD_FLAGS ${LOC_BUILD_FLAGS} PARENT_SCOPE)
+        elseif(KEY STREQUAL "vulkan")
+            list(GET ITEM_LIST 1 VULKAN_STR)
+            string(TOUPPER ${VULKAN_STR} VULKAN_UPPER)
+            if(VULKAN_UPPER STREQUAL "TRUE")
+                set(ILLIXR_VULKAN ON PARENT_SCOPE)
+                add_compile_definitions(ILLIXR_VULKAN)
+            endif()
```

Comment:
Why is this here? I see no profiles that specify 'vulkan' as an option.

Reply:
This should be addressed now (with the latest commits)